clipper_admin/clipper_admin/clipper_admin.py

```diff
@@ -91,11 +93,26 @@ def start_clipper(
             compability and preserve the expected behavior of the system.
         cache_size : int, optional
             The size of Clipper's prediction cache in bytes. Default cache size is 32 MiB.
-
+        config_file: str(optional)
+            File for the configuration of the application or model
         Raises
         ------
         :py:exc:`clipper.ClipperException`
         """
+        if(config_file is not None):
+           with open(config_file, "r") as stream:
+                config = yaml.load(stream)
+                for key in config:
+                    name = config[key]["name"]
+                    version = config[key]["version"]
+                    input_type = config[key]["input_type"]
+                    obj_type = config[key]["obj_type"]
+                    config_image = config[key]["image"]
```

Comment:
Applications will not need a configuration image, so we should only check for this key if we're parsing a **model** configuration.